dali/base/dautils.cpp

```diff
@@ -149,6 +149,11 @@ IPropertyTree * findDropZonePlane(const char * path, const char * host, bool ipM
     return nullptr;
 }
 
+bool isPathInPlane(IPropertyTree *plane, const char *path)
+{
+    return !isEmptyString(path) && startsWith(path, plane->queryProp("@prefix"));
```

Comment:
As it stands:

> false == isPathInPlance(plane, "")
> false == isPathInPlance(plane, nullptr)

Hopefully it would not be called with an empty path, but I think if it is, it makes more sense to return true.

Reply:
Changed